crates/sozo/src/commands/events.rs

```diff
@@ -38,6 +47,15 @@ pub struct EventsArgs {
 
 impl EventsArgs {
     pub fn run(self, config: &Config) -> Result<()> {
+        let target_dir = config.target_dir().path_existent().unwrap();
+        let manifest_path = target_dir.join(config.profile().as_str()).join("manifest.json");
```

Comment:
we should probably move `manifest.json` to being configurable (for all cli) in the future

Reply:
`Scarb.toml` file from which we can derive this `manifest.json` path is already configurable.

do we expect ppl to run `sozo events` without having access to the project file but just the `manifest.json`?

also scarb also calls their config file a manifest file so its confusing to talk about (in `SozoArgs` we refer to `Scarb.toml` file with env variable `DOJO_MANIFEST_FILE`), so lets use word scarb manifest for `Scarb.toml` and dojo manifest for `manifest.json`

Reply:
No, the idea is that we may have multiple manifest files for a single scarb.toml in the future. That is out of scope of this PR though. Just mentioning it since it came to mind